docs/basics/orders.rst

```diff
@@ -137,4 +137,44 @@ In both cases, the ``@0x/protocol-utils`` package simplifies generating these si
 
 The Orderbook 
 =======================
-Orders are shared through a decentralized and permissionless network, called `0x Mesh <https://0x.org/mesh>`_. The simplest way to post and discover orders is through `0x API <https://0x.org/api>`_. See `this guide <https://0x.org/docs/guides/market-making-on-0x>`_ tailored for Market Makers.
\ No newline at end of file
+Orders are shared through a decentralized and permissionless network, called `0x Mesh <https://0x.org/mesh>`_. The simplest way to post and discover orders is through `0x API <https://0x.org/api>`_. See `this guide <https://0x.org/docs/guides/market-making-on-0x>`_ tailored for Market Makers.
+
+Orders are usually represented as a JSON object off-chain. Below is a table represention and example of how orders should be formatted off-chain.
+
+JSON representation of RFQ Orders
+*********************************
+
+A ``RFQOrder`` should be serialized to JSON as following:
+::
```

Comment:
Ah I think this may be a gotcha on readthedocs, but I think you have to specify a language or it won't compile in the live version. Can just do `.. code-block:: solidity`

    
```

Reply:
Thanks! fixed